Intact.BuinessLogic/Services/RoomsService.cs

```diff
@@ -0,0 +1,91 @@
+using Intact.BusinessLogic.Data;
+using Intact.BusinessLogic.Data.Enums;
+using Intact.BusinessLogic.Data.Models;
+using Intact.BusinessLogic.Mappers;
+using Intact.BusinessLogic.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace Intact.BusinessLogic.Services;
+
+public interface IRoomsService
+{
+    Task<IEnumerable<Room>> GetAllAsync(CancellationToken cancellationToken);
+    
+    Task<int?> CreateAsync(Guid userId, string title, CancellationToken cancellationToken);
+    
+    Task<bool> GetAvailabilityAsync(int id, CancellationToken cancellationToken);
+    
+    Task<bool> JoinAsync(Guid userId, int id, CancellationToken cancellationToken);
+    
+    Task ExitAsync(Guid userId, int id, CancellationToken cancellationToken);
+}
+
+public class RoomsService(AppDbContext appDbContext, IProfilesService profilesService): IRoomsService
+{
+    private const int MaxPlayers = 8;
```

Comment:
I think we can do 2 for now, and in future we will add this as a room property when user creates it

Reply:
remade to 2 by one of the last commits